docs/docs/widgets/bounded-box.md

```diff
@@ -3,6 +3,10 @@ id: bounded-box
 title: Bounded Box
 ---
 
+Here's the updated version with the requested changes:
```

Comment:
Hey @bhojrajnarwae, you don't need to add this to the documentation. Please remove it.

Reply:
@RudraPratik30 I have made the required changes as per given instructions. please check it if everything is fine.